### ISSUE TITLE
SEM Pages - Legacy Layout

### ISSUE THREAD

**Issue by colinvlasak** [aemsites/stericycle-shared#564]
There are some styling issues with legacy layout SEM pages (Marketing pages with the light blue header and no nav or CTA in header)

Example: https://main--shredit--stericycle.aem.live/en-us/marketing/document-destruction-for-one-time-and-regular 

1) The header is sticky and it should not be. The sticky header should not apply to these pages. The new marketing pages (with navy header and nav should be sticky, as should main website header, but not this one).
2) Header logo is off on mobile. Should be left aligned to the margin like it is on our live site.
3) Inline form should be right under title/text. Above the one time and regular boxes on mobile. See live site.
4) Footer is full width but rest of content is not. Need to have separate footer for legacy SEM and new SEM pages as they have different margins. 

![image](https://github.com/user-attachments/assets/888f9ae6-d375-465f-9984-f7b7cd88bdba)

**Comment by MarquiseRosier** [comment]
Hi Colin, correct me if i'm wrong, but if the new marketing pages are pretty recent additions to the site; as discussed with @rwcsrichardspencer  we are not able to take on additional work in the form of site changes, updates that have occurred within the past month. Can you earmark any new work, changes to content and code that emerge, to be completed for the developers at Hero Digital? :)

**Comment by colinvlasak** [comment]
@MarquiseRosier page looks much better!

it looks like 3 was overlooked though. Can we make it so that the form comes after the text on mobile? Ahead of the One-Time Shredding Service and Regular Shredding With ProtectPLUS sections? This is how our pages are structured on the live site. Desktop layout should remain the same.

**Comment by MarquiseRosier** [comment]
@colinvlasak I'm sorry aha can you give some images of what you mean on point 3

**Comment by colinvlasak** [comment]
@MarquiseRosier

On mobile the form needs to be higher up on the page. On our current site it's the first thing after the title. On EDS there are the two service boxes on top of it. We have done testing in the past that showed having the form higher up on mobile for SEM pages drove more conversions.

Ideal state would be the form comes right after the intro text and before the different service boxes.

Current state EDS vs. Live

![image](https://github.com/user-attachments/assets/2dbcf056-2053-4ef0-9db1-d74a71a9bd71)

Ideal State

![image](https://github.com/user-attachments/assets/3e71826e-ee0f-40a3-8564-d92e6fd25132)

**Comment by ravise5** [comment]
@MarquiseRosier these are marketing location pages where we use a sidebar, I have fixed this issue already for two-column grid layout.

**Comment by MarquiseRosier** [comment]
@ravise5 yes you've fixed the issue; thank you, can you now implement your fix here?

**Comment by ravise5** [comment]
@MarquiseRosier So a lot of pages we are using page-sidebar in marketing template for embedding forms so what I meant here is in your page-sidebar implementation you need to make sure its shown above content in mobile view.  

cc: @jalagari

**Comment by colinvlasak** [comment]
@MarquiseRosier can you make this one a priority? A lot of our paid search pages use this layout

**Comment by MarquiseRosier** [comment]
Hello @ravise5 @jalagari I'd love to hop on a call about this issue. I'm perhaps a bit confused about why, we have 2 solutions for the same thing. If you're saying, you created an implementation of the sidebar that fixes this, A) why are we using the template we created purely to help you folks embed forms easily, and B) if you're new solution is fixing this legacy template, why are we not ensuring we use your solution here?

**Comment by MarquiseRosier** [comment]
It seems like it's your work to go through and replace these? Am I thinking about this incorrectly?

**Comment by ravise5** [comment]
@MarquiseRosier There are 60 marketing-location-pages currently that use this implementation, which will have to be manually changed.  Making these changes might also impact the styling of all these pages and that is something we don't want when go-live is so close. 

The new approach is only for some complicated pages where your earlier implementation was not working. 
cc: @jalagari

**Comment by MarquiseRosier** [comment]
thanks @ravise5 okay, i'll get a fix in soon!

**Comment by colinvlasak** [comment]
@MarquiseRosier when can we have this fixed and rolled out? 75% of the pages that are still pending QA verification are a result of this issue.
